.github/workflows/wpt_e2e.yml

```diff
@@ -23,6 +23,10 @@ jobs:
           cache: npm
       - name: Install and build npm dependencies
         run: npm ci
+      - name: Setup dirs
+        run: |
+          mkdir -p logs
+          mkdir -p wpt-metadata/chromeDriver
```

Comment:
These can be inlined:

mkdir -p logs wpt-metadata/chromeDriver

Reply:
done